packages/manager/modules/emailpro/src/dashboard/emailpro.controller.js

```diff
@@ -207,6 +207,12 @@ export default /* @ngInject */ function EmailProCtrl(
           loadPtrv6Tooltip($scope.exchange);
         }
       })
+      .then(async () => {
+        if ($scope.exchange.isMXPlan) {
+          const { isZimbra } = await EmailPro.retrieveMxPlan();
+          $scope.isMxPlanMigrated = isZimbra;
+        }
+      })
```

Comment:
As you don't use the previous call back, there is no need to chain it in the next `then` callback. You can merge it with a previous one